### PR TITLE
Add and style animation stress text

### PR DESCRIPTION
**Pull request by tealefristoe**
- Displays different text in the Stress Input area based on the Stress Level control.
- Student entered examples are saved when Stress Level is toggled.

NOTE: I made this branch from a development branch rather than main. It looks like it includes all of the changes from that previous branch as well as the changes for this branch, so it looks much more substantial than it actually is. Good learning experience for me!

### CODE REVIEW

File: cypress/e2e/workspace.test.ts [hunk 1]
```diff
@@ -74,13 +74,17 @@ context("Test the overall app", () => {
...
       const exampleText = "example text";
+      const exampleText2 = "example2";
       getTextArea().type(exampleText);
       getTextArea().blur();
       getTextArea().should("have.text", exampleText);
       getStressControl().click();
       getTextArea().should("have.text", "");
+      getTextArea().type(exampleText2);
       getStressControl().click();
       getTextArea().should("have.text", exampleText);
```
Comment (bacalj) [inline cypress/e2e/workspace.test.ts:85]: doesn't this line 85 (`getTextArea().type(exampleText2);`) need to be deleted?

**Review by bacalj — APPROVED** [review]
👍 looks good - one item I wasn't sure looked right.   Also do we care about potentially longer text?

File: cypress/e2e/workspace.test.ts (hunk 1 above)
Comment (tealefristoe) [reply cypress/e2e/workspace.test.ts:85]: That line writes different text into the high stress box, which was previously empty, so we can make sure it saves. I added it to test a single line of code :D

**Comment by tealefristoe** [comment]
Good question about longer text. I think the intention is that the box will scroll if there's more text than will fit in it.

**Pull request merged**